docs/faq.rst

```diff
@@ -0,0 +1,102 @@
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    In our :ref:`user guide <user_guide>` we provide links to the papers describing
+    Fairlearn's algorithms.
+    For a higher level view of the subject, please see the :ref:`further reading <further_reading>`
+    section.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
+
+    To elaborate, information is often redundantly encoded in datasets, and machine learning
+    algorithms *will* ferret out these links (it is what they are designed to do).
+    For example, in the US, the ZIP code where a person lives is well correlated with their
+    race.
+    Even if the model is not provided with race as a feature, the model will pick up on it
+    implicitly via the ZIP code (and other features).
+    Worse, without having the race available in the dataset, it is not possible to determine
+    whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+The model is unfair because the data are biased. Isn't it better to get better data?
+    Yes, but gathering unbiased data might not always be practical.
+    Fairlearn and similar packages offer techniques to redistribute harms between subgroups
+    in the data.
+
+Won't making a model fairer reduce its accuracy?
+    Any technique which reduces disparity in some metric is most likely to impact other metrics.
+    However, we have found empirically that it is usually possible to obtain substantial reductions
+    in disparity (as measured by some suitable metric) at minimal cost in accuracy.
+    Indeed, for certain subgroups in the data, accuracy may be improved dramatically.
+
+Fairlearn questions
+-------------------
+
+Can Fairlearn make my model fair?
+    Naive use of Fairlearn is not going to result in fairer models.
+    Indeed, when used incorrectly, Fairlearn could produce model which is *less* fair.
+
+    What Fairlearn can do is generate a model which has lower disparity as measured by
+    a user-specified metric.
+    Whether this makes the model fairer is outside the remit of Fairlearn - or indeed of
+    machine learning in general.
+    To determine whether the new model is fairer, the whole system of which it is a part
+    needs to be considered in terms of the societal context in which it is used.
+
+What sort of fairness related harms can Fairlearn address?
+    Fairlean concentrates on group fairness.
+    That is, obtaining the minimum disparity on some metric evaluated on
+    specified subgroups in the data.
+    This is certainly not the only way of looking at fairness, though.
+    For example, there is the concept of individual fairness (where decisions are evaluated
+    on the level of individuals rather than groups) and counter-factual fairness (e.g. does the
+    decision change if an individual's gender is changed from male to female).
```

Comment:
Shall we also point out aspects of fairness that Fairlearn does not address? Our styleguide mentions justice, due process, remedying historical societal injustices.